src/compiler/crystal/syntax/parser.cr

```diff
@@ -4914,6 +4914,9 @@ module Crystal
         exps << parse_op_assign
         if @token.type == :","
           next_token_skip_space_or_newline
+        else
+          skip_space_or_newline
+          check :")"
```

Comment:
Maybe this fix seems funny. But these lines are derived from usual arguments parsing ([here](https://github.com/crystal-lang/crystal/blob/b01ce093ece9cbad39e081015511f8e5e4255b4d/src/compiler/crystal/syntax/parser.cr#L3397-L3400))